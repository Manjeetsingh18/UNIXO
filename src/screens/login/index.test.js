import 'react-native';
import React from 'react';
import Login from '.';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly across screens', () => {
  const login = renderer.create(<Login />);
  expect(login).toMatchSnapshot();
});
