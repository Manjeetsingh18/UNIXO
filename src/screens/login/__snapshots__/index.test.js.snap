// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly across screens 1`] = `
<View
  style={
    {
      "flex": 1,
    }
  }
>
  <Text>
    Login
  </Text>
</View>
`;
